packages/store-api/src/platforms/vtex/clients/is/index.ts

```diff
@@ -0,0 +1,90 @@
+import { fetchAPI } from '../common'
+import type { Options } from '../..'
+import type { ProductSearchResult } from './types/ProductSearchResult'
+import type { AttributeSearchResult } from './types/AttributeSearchResult'
```

Comment:
`is` it's not a good name, maybe you can use `intelligent-search`.

Reply:
what about search?